modules/postman-templating/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "postman-templating",
+  "version": "1.0.0",
+  "description": "Templating for Postman",
+  "main": "build/index.js",
+  "scripts": {
+    "start": "tsc && node build/index.js",
+    "build": "rimraf build && tsc",
+    "prepare": "npm run build",
```

```suggestion
    "preinstall": "npm run build",
```

`prepare` doesn't run when _another_ module installs `postman-templating` as a dependency. `preinstall` will ensure that another package installing this will run the build step. Found the info at [npm docs](https://docs.npmjs.com/misc/scripts). Might help us avoid the manual `npm install`.